### ISSUE TITLE
Previously succeeding pipeline build fails with 1.3.7, unable to find openssl

### ISSUE THREAD

**Issue by doticatto** [mehcode/python-xmlsec#140]
```gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -DXMLSEC_CRYPTO=openssl -DXMLSEC_CRYPTO_DYNAMIC_LOADING=1 -D__XMLSEC_FUNCTION__=__FUNCTION__ -DXMLSEC_NO_SIZE_T -DXMLSEC_NO_GOST=1 -DXMLSEC_NO_XKMS=1 -DXMLSEC_DL_LIBLTDL=1 -DMODULE_NAME=xmlsec -DMODULE_VERSION=1.3.7 -I/usr/include/xmlsec1 -I/usr/include/libxml2 -I/tmp/pip-build-env-5qxk4kx4/normal/lib/python3.6/site-packages/lxml/includes -I/tmp/pip-build-env-5qxk4kx4/normal/lib/python3.6/site-packages/lxml -I/tmp/pip-build-env-5qxk4kx4/normal/lib/python3.6/site-packages/lxml/includes/__pycache__ -I/tmp/pip-build-env-5qxk4kx4/normal/lib/python3.6/site-packages/lxml/includes/libexslt -I/tmp/pip-build-env-5qxk4kx4/normal/lib/python3.6/site-packages/lxml/includes/libxslt -I/tmp/pip-build-env-5qxk4kx4/normal/lib/python3.6/site-packages/lxml/includes/libxml -I/builds/navigate-central-auth/navigate-central-auth/saml2/include -I/var/lang/include/python3.6m -c /tmp/pip-install-0iy2_hnb/xmlsec/src/main.c -o build/temp.linux-x86_64-3.6/tmp/pip-install-0iy2_hnb/xmlsec/src/main.o -g -std=c99 -fPIC -fno-strict-aliasing -Wno-error=declaration-after-statement -Werror=implicit-function-declaration -Os
   /tmp/pip-install-0iy2_hnb/xmlsec/src/main.c: In function ‘PyXmlSec_GetCryptoLibName’:
   <command-line>:0:15: error: ‘openssl’ undeclared (first use in this function)
   /tmp/pip-install-0iy2_hnb/xmlsec/src/main.c:33:49: note: in expansion of macro ‘XMLSEC_CRYPTO’
        const xmlChar* cryptoLib = (const xmlChar*) XMLSEC_CRYPTO;
                                                    ^
   <command-line>:0:15: note: each undeclared identifier is reported only once for each function it appears in
   /tmp/pip-install-0iy2_hnb/xmlsec/src/main.c:33:49: note: in expansion of macro ‘XMLSEC_CRYPTO’
        const xmlChar* cryptoLib = (const xmlChar*) XMLSEC_CRYPTO;
                                                    ^
   error: command 'gcc' failed with exit status 1
```

All of the needed packages in the readme are installed, and this exact command worked previously. Freezing at 1.3.3 works properly.

python xmlsec version: 1.3.7
libxmlsec1 version: 1.2.20-7.4
libxml2 version: 2.9.1-6.3.52

pkg-config output: 
```-DXMLSEC_CRYPTO=\"openssl\" -DXMLSEC_CRYPTO_DYNAMIC_LOADING=1 -D__XMLSEC_FUNCTION__=__FUNCTION__ -DXMLSEC_NO_SIZE_T -DXMLSEC_NO_GOST=1 -DXMLSEC_NO_XKMS=1 -DXMLSEC_DL_LIBLTDL=1 -I/usr/include/xmlsec1 -I/usr/include/libxml2  ```

**Comment by hoefling** [comment]
Sigh. I knew I will forget something. @doticatto can you please try installing the fixed version and check whether the error is resolved? I have made an upload to TestPyPI, you can try it out via

```sh
$ pip install xmlsec==1.3.8a1 --extra-index-url=https://test.pypi.org/simple
```

If you can confirm the issue resolved, I will prepare another release later today.

**Comment by llostris** [comment]
@hoefling We have had the same issue today, and using `xmlsec==1.3.8a1` resolved the problem.

**Comment by doticatto** [comment]
This fix resolved the issue.

**Comment by hoefling** [comment]
I have released [1.3.8](https://pypi.org/project/xmlsec/1.3.8/) which should resolve the issue. Sorry for the noise!

**Comment by doticatto** [comment]
Thank you for addressing this quickly
